src/connector/src/parser/mod.rs

```diff
@@ -759,10 +761,15 @@ async fn into_chunk_stream_inner<P: ByteStreamSourceParser>(
             if let SourceMeta::DebeziumCdc(msg_meta) = &msg.meta {
                 let lag_ms = process_time_ms - msg_meta.source_ts_ms;
                 // report to promethus
-                GLOBAL_SOURCE_METRICS
-                    .direct_cdc_event_lag_latency
-                    .with_guarded_label_values(&[&msg_meta.full_table_name])
-                    .observe(lag_ms as f64);
+                let full_table_name = msg_meta.full_table_name.clone();
+                let direct_cdc_event_lag_latency = direct_cdc_event_lag_latency_metrics
+                    .entry(full_table_name)
+                    .or_insert(
+                        GLOBAL_SOURCE_METRICS
+                            .direct_cdc_event_lag_latency
+                            .with_guarded_label_values(&[&msg_meta.full_table_name]),
```

Comment:
since the label is determined during actually reading the message, so a hashmap to hold it till the end